app/views/admin/tags/_form.html.erb

```diff
@@ -48,6 +48,7 @@
     <%= f.label :wiki_body_markdown, class: "crayons-field__label" %>
     <%= f.text_area :wiki_body_markdown, value: tag.wiki_body_markdown, class: "crayons-textfield" %>
   </div>
+  <div class="crayons-notice crayons-notice--warning mt-3">We will soon update how tags are being displayed. Please define tag's primary color in "Bg color hex" field - the other one ("Text color hex") will be deprecated.</div>
```

Comment:
I guess it is already deprecated? In that it doesn't currently do anything?

Reply:
I want to keep it for folks who will want to update tags colors once we ship this change. So they can more easily swap colors if they want to (some ppl used primary color for text instead of background). So I'll give it few weeks as an adaptation period. Plus I'd prefer to remove this field alongside backend changes (so we also don't store it in the DB).